pkg/config/config.go

```diff
@@ -164,7 +164,8 @@ type (
 	}
 
 	seed struct {
-		Enabled bool `toml:"enabled"`
+		Enabled bool     `toml:"enabled"`
+		Path    []string `toml:"path"`
```

Comment:
```suggestion
		SqlPaths    []string `toml:"sql_paths"`
```

May be call this sql path to be more specific.